python_modules/libraries/dagster-k8s/dagster_k8s/job.py

```diff
@@ -944,6 +959,13 @@ def construct_dagster_k8s_job(
                             dagster_labels, user_defined_job_labels, job_config.labels
                         ),
                     },
+                    {
+                        "owner_references": [
+                            owner_reference.to_dict() for owner_reference in owner_references
+                        ]
+                    }
+                    if owner_references
+                    else {},
```

Comment:
i cannot imagine this will be common at all, but you should probably append this to any owner_references that are set in user_defined_job_metadata instead of overwriting them (similar to how user_defined_job_labels is handled just above)